CodeforNepal/nepalmap2021
# add blog feature in django admin

# Details of PR
- create blog admin app
- create blog base template
- new migrations

# Steps to test
- checkout the branch
- create a superuser: `./manage.py createsuperuser` and follow the prompts (this will only affect your local system)
- run the migration `./manage.py migrate`
- run the local server `./manage.py runserver`
- login to django admin using the superuser credentials `/admin`
- you should be able to see a section for Blog Entries and be able to add, edit and delete blog entries

# Screenshots
![Screenshot from 2022-08-07 17-47-28](https://user-images.githubusercontent.com/191368/183313982-86a3fe09-ce68-41f5-8dc2-98141f78cb6e.png)

![Screenshot from 2022-08-07 17-48-36](https://user-images.githubusercontent.com/191368/183313995-dc9d3a16-696d-4aaf-a1da-42a49f37559e.png)

![Screenshot from 2022-08-07 17-48-49](https://user-images.githubusercontent.com/191368/183314005-035f5084-534a-47bc-b246-d6f668d250b5.png)
